.github/workflows/ci.yml

```diff
@@ -53,6 +53,7 @@ jobs:
       - test_app
       - test_frontend
     runs-on: ubuntu-latest
+    if: github.ref == 'refs/heads/main'
```

Comment:
As this project still uses `master` as the default branch name, this `if` doesn't have any effects. I'll fix this by myself.